src/main/java/raisetech/student/management/data/ErrorStudentsCourse.java

```diff
@@ -0,0 +1,13 @@
+package raisetech.student.management.data;
+
+import java.time.LocalDate;
+
+public class ErrorStudentsCourse {
+
+  private String courseId = "0";
```

Comment:
デフォルトの値必要ですか？
```suggestion
  private String courseId;
```
でいいような。